presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/EvaluateZeroDistinctLimit.java

```diff
@@ -1,4 +1,3 @@
-
 /*
```

Comment:
deleting the newline should be part of the commit that adds this class

Reply:
my apology could you please elaborate a little bit? I am unsure what you mean by this "deleting this newline?

Reply:
If you look at the changes commit by commit, you'll see that the commit "Avoid planning unnecessary sort" has a change that removes an extra line at the start of EvaluteZeroLimit.java.  This should be done in the previous commit where the class is added (i.e. don't have the extra line in the first place)

Reply:
understood, fixed now in first commit